sdks/python/apache_beam/transforms/__init__.py

```diff
@@ -23,3 +23,6 @@
 from apache_beam.transforms.ptransform import *
 from apache_beam.transforms.timeutil import *
 from apache_beam.transforms.util import *
+
+
+del typehints
```

Comment:
I'm going to try rebasing this on the fixes from transforms; hopefully I won't need it anymore.